### ISSUE TITLE
react-addons-css-transition-group deprecated in React 16+

### ISSUE THREAD

**Issue by frankolson** [chadly/react-bs-notifier#27]
Getting the following warning when resolving this package:

```bash
warning react-bs-notifier > react-addons-css-transition-group@15.5.2: This package is deprecated and will no longer work with React 16+. We recommend you use CSSTransitionGroup from 'react-transition-group' instead.
```

Proposing the switch from [react-addons-css-transition-group](https://www.npmjs.com/package/react-addons-css-transition-group) (deprecated in React 16+) to [react-transition-group](https://github.com/reactjs/react-transition-group)

**Comment by chadly** [comment]
Yes, this is something I haven't gotten around to yet.

PRs welcome 😄

**Comment by chadly** [comment]
Fixed via #33. On npm as `v4.4.0`
